### PR TITLE
update v1.2.5

### PR DESCRIPTION
**Pull request by zerinf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,27 +48,30 @@ The CryoSieve package depends on the following libraries:
...
 
-To streamline this process, we suggest preparing a conda environment with the following command:
+For CUDA version <= 11.7:
 ```
-conda create -n CRYOSIEVE_ENV python=3.10 cupy=10.2 cudatoolkit=10.2 pytorch=1.12.1=py3.10_cuda10.2_cudnn7.6.5_0 -c conda-forge -c pytorch
+conda create -n CRYOSIEVE_ENV python=3.8 cudatoolkit=10.2 cupy=10.0 pytorch=1.10 -c pytorch -c conda-forge
 ```
+Please note that this command is tailored for CUDA version 10.2. For a different CUDA version, you should adjust the `cudatoolkit` version accordingly. You can also modify the versions of Python, CuPy, and PyTorch based on your requirements, while ensuring they meet the minimal requirements of CryoSieve.
 
-This command is specifically for a CUDA environment version 10.2. If your CUDA environment is higher than 10.2, adjust the command based on the suitable variants and versions recommended by the [CuPy](https://cupy.dev) and [PyTorch](https://pytorch.org) developers for your specific CUDA environment.
```
Comment (mxhulab) [inline README.md:64]: Avoid using "you" or "your" in the text. Instead, opt for the imperative mood.

比如这里，请修改为
Note that this command is tailored for CUDA version 10.2. To accommodate a different CUDA version, adjust the `cudatoolkit` version accordingly. Modify the versions of Python, [CuPy](https://cupy.dev), and [PyTorch](https://pytorch.org) based on requirements, ensuring compatibility with the minimal requirements of CryoSieve.

File: README.md [hunk 2]
```diff
@@ -48,27 +48,30 @@ The CryoSieve package depends on the following libraries:
...
+For CUDA version >= 11.8:
+```
+conda create -n CRYOSIEVE_ENV python=3.10 cupy=12.0 pytorch pytorch-cuda=12.1 -c pytorch -c nvidia -c conda-forge
+```
+Please note that This command is tailored for CUDA environment version 12.1. For a different CUDA version, you should adjust `pytorch-cuda` version accordingly.
```
Comment (mxhulab) [inline README.md]: 这里建议修改为

Note that this command is tailored for CUDA environment version 12.1. For a different CUDA version, adjust the `pytorch-cuda` version accordingly.

**Review by mxhulab — CHANGES_REQUESTED** [review]
请稍微修改一下 README。其他没有问题，非常👍。

**Comment by zerinf** [comment]
这两处已修改。

**Review by mxhulab — APPROVED** [review]

**Pull request merged**